web/src/loading.ts

```diff
@@ -108,3 +108,14 @@ export function show_spinner($element: JQuery, $spinner: JQuery): void {
         height: span_height,
     });
 }
+
+export function hide_spinner($element: JQuery, $spinner: JQuery): void {
```

Comment:
Do these functions expect button elements? `$button_element` would be a much clearer name if so.

Also the name `show_button_spinner` seems more rational; though there is already a `show_button_spinner`.

I think ultimately we will likely want to rework all the callers of that function to share this code path. Maybe you can look at adding a follow-up commit that reworks the existing callers of `show_button_spinner` to use this new API (probably includes template changes) and then we can rename to `show_button_spinner` the surviving version.